pyccl/tracers.py

```diff
@@ -1,12 +1,40 @@
+r"""
+Tracers represent projected quantities that can be cross-correlated to get
+angular power spectra (see :func:`~pyccl.cells.angular_cl`). In the most
+general case, the angular power spectrum between two tracers is given by
+
+.. math::
+    C^{\alpha\beta}_\ell=\frac{2}{\pi}\int d\chi_1\,d\chi_2\,dk\,k^2
+    P_{\alpha\beta}(k,\chi_1,\chi_2)\,\Delta^\alpha_\ell(k,\chi_1)\,
+    \Delta^\beta_\ell(k,\chi_2),
+
+where :math:`P_{\alpha\beta}` is a generalized power spectrum (see
+:class:`~pyccl.pk2d.Pk2D`),
+and :math:`\Delta^\alpha_\ell(k,\chi)` is a sum over different contributions
+associated to tracer :math:`\alpha`, where every contribution takes the form:
+
+.. math::
+    \Delta^\alpha_\ell(k,\chi)=f^\alpha_\ell\,W_\alpha(\chi)\,
+    T_\alpha(k,\chi)\,j^{(n_\alpha)}_\ell(k\chi).
+
+Here, :math:`f^\alpha_\ell` is an :math:`\ell`-dependent **prefactor**,
+:math:`W_\alpha(\chi)` is the **radial kernel**, :math:`T_\alpha(k,\chi)`
+is the **transfer function**, and :math:`j^{(n)}_\ell(x)` is a
+generalized version of the **spherical Bessel functions**. Descriptions
+of each of these ingredients, and how to implement generalised ``Tracer`` s
```

Comment:
Why do you use here 
```
``Tracer``
```
instead of
```
:class:`Tracer`
```
?

Reply:
Done. We didn't really need either of them here.